tutorial/part2/chapter6.md

```diff
@@ -90,101 +92,93 @@ For this reason, we recommend importing the data before initialising the app, ri
+    'Country Code': ['US','NO','IT','SE','FR']
+})
 
 # Initialise app
 app = Dash(__name__)
 ```
 
-Then, some mock-up data can be added using a dictionary:
-```
-test_data = pd.DataFrame({'Country':['United States','Norway','Italy','Sweden','France'],
-                         'Country Code':['US','NO','IT','SE','FR']})
 ```
 The dictionary keys `Country` and `Country code` will represent the column names of the pandas dataframe.
+```
 
 ```{note}
-If you need access to pre-built data sets to test your code, consider using the Plotly Express built-in data, such as Gapminder or [other data](https://plotly.com/python-api-reference/generated/plotly.express.data.html), which can be added to your app with these commands:
-
-df = px.data.gapminder()
+If you need access to pre-built data sets to test your code, consider using the Plotly Express built-in data, such as Gapminder or [other of of the many other dataset in plotly express](https://plotly.com/python-api-reference/generated/plotly.express.data.html). To load in the Gapminder data we could use the following line of code: `df = px.data.gapminder()`.
 ```
 
-## 6.2 Uploading data into an app
-Whether our data is located on our computer or on the internet, we need to read the data from the source and convert it into a pandas dataframe. There are several ways of uploading data into our app; we will focus on these methodologies, as they are commonly used by Dash-app creators:
+## 6.2 Reading in data to use in Dash
 
-- reading data from files on our computer (.xls, .csv, .json)
-- reading data from a URL
+Data exists in multiple formats (e.g. `.csv`, `.json`, and `.xlsx`) and could be located on our computer or on the internet. Here we will use the powerful pandas library to show how you can read in data from various sources before using it in your app.
 
 ### 6.2.1 Reading data from files on your computer
 
 #### Excel files
-Let's see an example of how to upload an Excel file - extracted from the Gapminder data - into a dataframe. 
-The file we'll be using is available here [data_01](https://github.com/open-resources/dash_curriculum/blob/main/tutorial/part2/ch6_files/data_01.xlsx): follow the link and click on "Download"; find the folder where the file was downloaded and copy the path. Note that you will need to `pip install openpyxl` into your terminal in order to work with `.xlsx` files
+
+Let's see an example of how to read in an Excel file into a dataframe.
+The file we'll be using is available here [data_01](https://github.com/open-resources/dash_curriculum/blob/main/tutorial/part2/ch6_files/data_01.xlsx): follow the link and click on "Download"; find the folder where the file was downloaded and copy the path. Note that you will need to `pip install openpyxl` into your terminal in order to work with `.xlsx` files.
 
 ```
 import pandas as pd
+
+
+# Your file path might look different
 filepath = r'C:\Users\YourUser\Downloads\data_01.xlsx'
```

Comment:
We should format paths so that they are operating system agnostic or explain how to do it on different OSes. Can't you just `/` everywhere nowadays?